### PR TITLE
cleanup examples according to upcoming code guidelines

### PR DESCRIPTION
**Pull request by FloVanGH**

### CODE REVIEW

File: examples/7guis/cells.slint [hunk 1]
```diff
@@ -58,15 +58,16 @@ export component MainWindow inherits Window {
...
                     edited => {
                         cell = { value: self.text.to-float(), formula: self.text };
                     }
                     accepted => {
                         root.active-cell = { r: -1, c: -1};
                     }
+
+                    visible: is-active;
+                    width: 100%;
+                    height: 100%;
```
Comment (ogoffart) [inline examples/7guis/cells.slint:70]: not sure about this part of the guidelines that put callbacks connection before properties

Comment (FloVanGH) [reply examples/7guis/cells.slint:70]: I think that's consistent with properties the rest of the callbacks and function definitions.

Comment (FloVanGH) [reply examples/7guis/cells.slint:70]: image

**Comment by FloVanGH** [comment]
New order (according to new agreements)

1. `public` properties
1.1 `in` properties
1.2 `out` properties
1.3 `in-out` properties
2. `callbacks`
3. `public` functions
4. property bindings (geometry first x, y, width, height, ...)
5. `states`
6. `animations`
7. `private` properties
8. `private` functions
9. `children`

**Review by tronical — APPROVED** [review]

**Comment by FloVanGH** [comment]
I merge this now, because it cleanup the code and make it easier to adjust for the final best practices guidelines.  I will create an issue to track the state of documentation of the best practices as followup.

**Pull request merged**